gnosis/safe-react
# Rm redundant balances requests

## What it solves
Resolves #2239

## How this PR fixes it
We load the balances on safe load, and fetch them every 15 seconds afterwards.
No need to fetch them again on the Balances tab.

## How to test it
* Open a safe, make sure the balances are fetched
* Switch to another safe, balances should be fetched again
* Wait for 15 seconds, observe another request to update the balances

## Screenshots
Before | After
------ | ------
<img width="280" alt="Screenshot 2021-08-27 at 10 45 48" src="https://user-images.githubusercontent.com/381895/131099876-05d1ae6f-1dc0-4b05-aacd-0352bd692ff7.png"> | <img width="294" alt="Screenshot 2021-08-27 at 10 45 56" src="https://user-images.githubusercontent.com/381895/131099889-e438b718-12c9-4257-814e-1f673144dbb0.png">
